packages/core/contracts/insured-bridge/BridgePool.sol

```diff
@@ -75,12 +75,12 @@ contract BridgePool is Testable, BridgePoolInterface, ExpandedERC20, MultiCaller
 
     // A Relay is linked to a L2 Deposit.
     struct RelayData {
-        uint256 relayId;
         RelayState relayState;
-        uint256 priceRequestTime;
-        uint64 realizedLpFeePct;
         address slowRelayer;
+        uint32 relayId;
+        uint64 realizedLpFeePct;
         address instantRelayer;
+        uint256 priceRequestTime;
```

Comment:
is it not better to put all the `uintn` next to each other in the `struct`? also, put. the `address` together? I know with params this saves gas but not sure about struct values?

Reply:
Yeah, I reordered them a little bit to try and 1) reduce the amount of space this takes up 2) to reduce the number of `delete` calls I need to make in the settlement function to clear a slot. There are probably multiple orderings that would do this, but we can't put the addresses together without causing solidity to create an extra slot I don't think.